### DISCUSSION TITLE
윤우님, 혜인님 책 제안 받습니다.

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#4] [General]
@hi-hyein @yoonucho 

사실 책 목록을 두고 선정해 보려고 했는데
그러면 왠지 제가 시켜서 읽게 되는 책이 될 것 같아서요.

동훈님 처럼 자신이 보고 싶다는 책이 있으면 거리낌 없이 제안해 주시면 좋겠습니다.
소프트웨어 장인은 혜인님 윤우님도 좋다고 하셨기 때문에 1 tier로 선정된 책이라
이렇게 책을 선정하는 게 좋다는 생각입니다.

모집 기한 까지 대략 2주 정도 남았는데
그 기간 안에 조금 고민해 보시고 책을 선정해서 알려주시면 좋을 것 같아요.

**Comment by hi-hyein** [comment]
## 책 제안  
🔔  **클린 코더** 추천합니다!

클린 코드도 추천하고 싶었는데 코드에 대한 이야기라 주제가 소프트웨어 -> 코드짜는 법이 되어버릴까봐
클린 코더로 제안드려봅니다~

---

@jongfeel 
🔔  그리고 날짜 상관없이 학술회 참석가능합니다! 참석자분들 스케쥴 가능하신 날로 정해지면 알려주심 될 것 같아요!
(어디다 알려드릴지 애매해서 책 추천겸 여기에..ㅎㅎ)

**Reply by jongfeel** [reply]
클린 코드도 나쁘지 않은 책인데 둘다 Robert C Martin 할아버지 책이라 저는 좋은 것 같습니다.
회사에도 제가 책을 구매해 놓은게 있어서 좋네요.

윤우님하고 동훈님만 좋다면 두번째 선정되는 책일 것 같습니다.
(이미지 클릭하면 알라딘 링크로 갑니다)

[![CleanCoder](https://image.aladin.co.kr/product/8661/93/cover500/8960778818_1.jpg)](http://aladin.kr/p/MJJw1)

**Reply by jongfeel** [reply]
@Joseph-Cha , @yoonucho 
두분 알람 드리는 용도로 댓글 추가로 달아 봅니다.

**Reply by Joseph-Cha** [reply]
좋아요 ㅎㅎ 
클린 코더도 많이 들어봤던 책이라 더 기대가 됩니다😀

**Reply by Joseph-Cha** [reply]
종필님의 얘기를 들어보니 자주 들었던 책은 **클린 코드**였군요 ㅎㅎ

**Reply by yoonucho** [reply]
저도 좋습니다! 예전부터 보고싶어서 킵해둔 책 중 하나네요 ㅋ

**Reply by yoonucho** [reply]
음 그럼 저는 ~~소프트웨어 스펙의 모든 것~~을 제안드려도 될까요?
북토크에 적합한지는 멘토님이 확인해주셔요! 
멘토님과 상의 후 다른 책을 다시 골라보겠습니다.

**Reply by hi-hyein** [reply]
윤우님이 추천해주신 소프트웨어 스펙의 모든 것 목차 보고왔는데 저는 너무 재밌을 것 같아요!

**Reply by jongfeel** [reply]
어느 부분은 괜찮다고 보는데, 모든 내용을 가지고 하기에는 조금 부적합하거나 아쉬운 부분도 있어 보여서 얘기 했습니다.
저는 어느정도 괜찮다고는 얘기했는데, 윤우님이 다른 책 고르신다고 하는 군요.

**Reply by yoonucho** [reply]
어랏 솔깃 그럼 그대로 진행할까요?ㅋ

**Reply by Joseph-Cha** [reply]
저도 목차랑 리뷰를 봤는데 저 같은 주니어 개발자에게는 조금 이른 감이 있는 내용이 될 것 같아요 ㅠ 아직 프로젝트를 기획하는 단계에 투입되지 않다 보니 구체적인 스펙을 작성하는 방법에 대해서는 경력이 좀 있는 시니어나 팀 리더 급 개발자 분들에게 더 도움이 되지 않을까 싶네요 ㅎㅎ (ps. 너무 솔직한 의견이라 상처가 되진 않을까 걱정이군요;;ㅠㅠ)

**Reply by yoonucho** [reply]
동훈님 말씀도 맞긴 합니다. ㅎㅎ 그래서 멘토님도 두분이 제안하신 책과 제가 말씀드린 책은 결이 좀 다르다고 하셔서  다른 책도 생각하고 있습니다 ㅎㅎ (상처안받았어요 ㅋ )

**Reply by Joseph-Cha** [reply]
그렇다면 다행이네요😄 결이 다른 것도 저는 좋다고 생각합니다~ 같이 고민해보죠 ㅎㅎ

**Reply by jongfeel** [reply]
고민하지 마시고 <소프트웨어 스펙의 모든 것> 결정하시죠.
빅픽쳐를 고려해 나중에 글로벌한 소프트웨어 회사에서 일하게 된다고 했을 때는 정말 도움이 되실 겁니다.

**Reply by yoonucho** [reply]
음 그럼 그렇게 할게요!ㅋ
